### PR TITLE
add json resolve for FI/FO

### PR DESCRIPTION
**Pull request by zpltys**

### CODE REVIEW

File: src/algorithm/graphSim.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package algorithm
```
Comment (bupt-wenxiaole) [inline src/algorithm/graphSim.go]: Please revert this file, this file does not include Graph Simulation algorithm

File: test_data/subgraph.json [hunk 2]
```diff
@@ -1,34 +1,41 @@
 {
```
Comment (bupt-wenxiaole) [inline test_data/subgraph.json:1]: if the Graph's node doesn't include attr, the function NewGraphJSON still works?

**Review by bupt-wenxiaole — COMMENTED** [review]
Please delete the debug info and blank lines then format the code, thanks for contribution

File: test_data/subgraph.json (hunk 2 above)
Comment (zpltys) [reply test_data/subgraph.json:1]: the attr is initialed to be 0, so it still works(set attr to 0)

**Review by bupt-wenxiaole — COMMENTED** [review]
OK, I will merge this PR, thanks

**Pull request merged**